packages/tradershub/src/features/cfd/constants.tsx

```diff
@@ -160,37 +160,37 @@ export const AppToContentMapper: TAppToContentMapper = {
         title: 'DerivX web',
     },
     ctrader_windows: {
-        icon: <WindowsIcon />,
+        icon: <WindowsIcon height={32} width={32} />,
```

Comment:
If the values are the same we'd better define variables and make use of that one.